src/ui/shared.js

```diff
@@ -431,7 +431,12 @@ window.exportPreferencesButton = async () => {
 
 window.exportPreferences = async () => {
   const storage = await browser.storage.local.get('preferences');
-  return JSON.stringify(storage.preferences, null, 2);
+  const exportedPreferences = {
+    version: browser.runtime.getManifest().version,
+    date: Date.now(),
```

Comment:
I'd rather use `new Date` instead of the timestamp. We probably won't get bitten by the [Year 2038 Problem](https://en.wikipedia.org/wiki/Year_2038_problem), but who knows.

Reply:
So I just learned that javascript uses [floats instead of integers](https://ecma-international.org/ecma-262/5.1/#sec-8.5) and thus isn't prone to the mentioned problem, so you can scratch that and leave as is if you prefer.